tasks/fetching.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
I don't see a patch on `tasks/main.yml` to include this file.
Did it got deleted during the numerous rebases ? 😖

Reply:
Exactly, I've fixed it + fast-forwarded to current master :)